lib/suspenders/generators/production/email_generator.rb

```diff
@@ -0,0 +1,30 @@
+require "rails/generators"
+
+module Suspenders
+  module Production
+    class EmailGenerator < Rails::Generators::Base
+      source_root File.expand_path(
+        File.join("..", "..", "..", "templates"),
+        File.dirname(__FILE__),
+      )
+
+      def smtp_configuration
+        copy_file "smtp.rb", "config/smtp.rb"
+
+        prepend_file "config/environments/production.rb",
+          %{require Rails.root.join("config/smtp")\n\n}
+      end
+
+      def use_smtp
+        config = <<-RUBY
+
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.smtp_settings = SMTP_SETTINGS
+        RUBY
```

Comment:
Would we be able to use a [squiggly heredoc](https://infinum.co/the-capsized-eight/multiline-strings-ruby-2-3-0-the-squiggly-heredoc) here to avoid the "inner" indentation?

Reply:
I thought about it, but I actually _do_ want the two spaces of indentation.